apisix/plugins/dynamic-upstream.lua

```diff
@@ -187,56 +187,51 @@ function _M.check_schema(conf)
 end
 
 
-local function generate_random_num(max_val)
-    local random_val = math.random(1,max_val)
-    return random_val
-end
-
-
 local operator_funcs = {
-    ["=="] = function(v2, ctx)
-        if ctx.var[v2[1]] and ctx.var[v2[1]] == v2[3] then
+    -- var value example: ["http_name", "==", "rose"]
```

Comment:
Got it.
Don't you think the following operator_funcs are similar? Can the operator be abstracted out?

Reply:
you can create a issue for this